src/handlers/thirdparty.ts

```diff
@@ -13,19 +18,16 @@ export function handleThirdPartyAdded(event: ThirdPartyAdded): void {
   thirdParty.maxItems = BigInt.fromI32(0)
   thirdParty.totalItems = BigInt.fromI32(0)
   thirdParty.isApproved = event.params._isApproved
-  thirdParty.managers = []
+  thirdParty.managers = new Array<string>()
 
-  let managers = thirdParty.managers
   let eventManagers = event.params._managers
 
   for (let i = 0; i < event.params._managers.length; i++) {
     let m = eventManagers.pop()
-    managers.push((m as Address).toHexString())
+    thirdParty.managers.push((m as Address).toHexString())
```

Comment:
I think that you broke the managers. https://thegraph.com/hosted-service/subgraph/decentraland/tpr-matic-mumbai?query=ThirdParties I see an empty array here! We should have ~ 10 if I remember ok

Reply:
Fixed! It didn't work the way it was because we can only `set` or `get` properties from the entities, we can't modify them. I think that the graph guys should move away from the JS getters and setters to avoid this issue.